dev/data-config_template.ttl

```diff
@@ -0,0 +1,49 @@
+#
+# RDF4J configuration template for a GraphDB repository
+#
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>.
+@prefix rep: <http://www.openrdf.org/config/repository#>.
+@prefix sr: <http://www.openrdf.org/config/repository/sail#>.
+@prefix sail: <http://www.openrdf.org/config/sail#>.
+@prefix graphdb: <http://www.ontotext.com/config/graphdb#>.
+
+[] a rep:Repository ;
+    rep:repositoryID "my_db" ;
```

Comment:
:cherries: since we just need to auto-replace this string during setup, should we rename this to something like "placeholder_name" or so to make it more clear that it'll be replaced by the DB_NAME ENV var?